scripts/index_congress_scores.py

```diff
@@ -110,17 +110,21 @@ def strip_accents(s):
 
 				# The reason to pop these values out of the row, is so we can assume all the data
 				# left in 'row' is voting data
-				name = row.pop(0)
+				first_name = row.pop(0)
+				last_name = row.pop(0)
 				legislator_id = row.pop(0)
 				state_district = row.pop(0)
 				party = row.pop(0)
 				total_score = row.pop(0)
 
+				if not re.search('^\d+%$', total_score):
+					total_score = 'N/A'
+
 				votes_total = 0
 				votes_agreed = 0
 				legislator_id = "aclu/elections-api/congress_legislator:%s" % legislator_id
 
-				if name == 'LEGEND:':
+				if first_name == 'LEGEND:':
```

Comment:
(This might be an issue for a separate PR!!) Should we not break the loop here if there's more legislators at the end of the spreadsheet who ended their terms early? 

I noticed that reindexing `details` and `scores` caused some score sessions for those legislators to be dropped from the endpoint, even though they currently exist on elections-stg and have been added to the bottom of the spreadsheet (like John McCain and Ron DeSantis).

Reply:
lol I totally didn't catch those at the bottom, thank you!